jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,88 @@
+describe('Index', function() {
+
+  var index = new Index();
+
+  describe('Read book data', function() {
+
+
+    it('should check if an Invalid Arguement is passed', function() {
+      expect(index.createIndex()).toBe('Invalid Arguement');
+    });
+
+    it('should check if the file passed is a valid file', function(done) {
+
+      index.createIndex('/test').catch(function(err) {
+        expect(err).toEqual(jasmine.any(Error));
+        expect(err.toString()).toEqual('Error: Not Found');
+        done();
+      });
+
+    });
+
+
+    it('should check if the file passed is a valid JSON and not empty', function(done) {
+
+      index.createIndex('/emptyBook.json').catch(function(err) {
```

Comment:
**Code Quality - Nitpick**

Use the path relative to the current directory  (e.g. `./emptyBook.json` or just `emptyBook.json`) instead of the  `/emptyBook.json` which is relative to the domain.

If the server is started in the root folder, `SpecRunner.html` will be at `http://localhost:8080/jasmine/SpecRunner.html`. In which case `/emptyBook.json` refers to `http://localhost:8080/emptyBook.json`. `./emptyBook.json` or just `emptyBook.json` would refer to `http://localhost:8080/jasmine/emptyBook.json` which is the correct location in this case